include/OpenSpaceToolkit/Astrodynamics/GuidanceLaw/QLaw.hpp

```diff
@@ -61,13 +63,14 @@ class QLaw : public GuidanceLaw
     struct Parameters
     {
         Parameters(
-            const Map<COE::Element, double>& anElementWeights,
+            const Map<COE::Element, Tuple<double, double>>& anElementWeightsMap,
             const Size& aMValue = 3,
             const Size& aNValue = 4,
             const Size& aRValue = 2,
+            const double& aBValue = 0.01,
             const Size& aKValue = 100,
-            const Length& minimumPeriapsisradius = Length::Kilometers(6578.0),
-            const double& aBValue = 0.01
+            const double& aPeriapsisWeight = 0.0,
```

Comment:
Alright, that actually answers of the question/suggestion I had to make this parameterizable ✅